charts/steward/templates/podsecuritypolicy-controller.yaml

```
@@ -0,0 +1,45 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: 00-steward-controller
```

Since this PSP belongs to both controllers I would name it:
```suggestion
  name: 00-steward-controllers
```
Also rename the `podsecuritypolicy-controller.yaml` file itself to `podsecuritypolicy-controllers.yaml`